src/components/engagement/handlers/update-project-status.handler.ts

```diff
@@ -53,4 +54,52 @@ export class UpdateProjectStatusHandler
       })
     );
   }
+
+  async handle({ previous, updated, updates, session }: ProjectUpdatedEvent) {
+    if (updated.status === ProjectStatus.Terminated) {
```

Comment:
Need to check `previous` as well. We don't want to run this code every time the project is updated after it's terminated.